src/Common/Geocoder.php

```diff
@@ -53,4 +53,15 @@ public function geocode(string $value): Collection;
      * @throws \Geocoder\Exception\Exception
      */
     public function reverse(float $latitude, float $longitude): Collection;
+
+    /**
+     * Lookup a location by it's provider-specific id.
+     *
+     * @param mixed $id
+     *
+     * @return Collection
+     *
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function lookup($id): Collection;
```

Comment:
```suggestion
    public function lookup(string $id): Collection;
```

I think it would be good to type hint this as `string`. I

Reply:
(and add this to all other places in the code)

Reply:
Adding this method here will break BC :/